python/pylibcudf/pyproject.toml

```diff
@@ -39,11 +39,13 @@ classifiers = [
 
 [project.optional-dependencies]
 test = [
+    "cupy-cuda12x>=12.0.0",
     "fastavro>=0.22.9",
     "hypothesis>=6.131.7",
     "mmh3",
     "nanoarrow",
-    "numpy>=1.23,<3.0a0",
+    "numba-cuda>=0.14.0,<0.15.0a0",
```

Comment:
This isn't aligned with #19604, does `branch-25.10` need to be merged in?

Reply:
Done, thanks.